theme.json

```diff
@@ -97,6 +97,63 @@
 			],
 			"units": ["%", "px", "em", "rem", "vh", "vw"]
 		},
+		"typography": {
+			"defaultFontSizes": false,
+			"fluid": true,
+			"fontSizes": [
+				{
+					"fluid": false,
+					"name": "Extra Small",
+					"size": "14px",
+					"slug": "x-small"
+				},
+				{
+					"fluid": {
+						"max": "18px",
+						"min": "18px"
+					},
+					"name": "Small",
+					"size": "1rem",
+					"slug": "small"
+				},
```

Comment:
If this is meant to be fluid, and 1rem is the size, should the min be 16px? Now both min and max are 18px which I believe may just be a typo?

Reply:
Sounds like it.